### ISSUE TITLE
4.2.2 > 4.4.0 Error 400 Bad request after update, but syncs OK

### ISSUE THREAD

**Issue by RetroGameTalk** [10up/ElasticPress#3253]
### Describe your question

![sync_ok](https://user-images.githubusercontent.com/8179611/211437619-7cf736db-46f7-4539-a160-d17dd2ab609f.PNG)
![server](https://user-images.githubusercontent.com/8179611/211437687-e1f510bd-ac87-4407-b040-78a1da787f5c.PNG)
I don't know what to do, health doesn't show errors.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felipeelia** [comment]
@MediaMaquina, ElasticPress 4.4.0 has a new Status Report page that logs up to 5 failed queries during the last 24 hours. There you should find the reason why that failed and possibly some hints on what should be done to fix it. Also, if you install the Debug Bar ElasticPress plugin, it will create a new view in Query Monitor with all the fired requests to ES and their responses.

Do you mind doing one of those two things and getting back to us with the reason why that query is failing? Thanks

**Comment by RetroGameTalk** [comment]
Hello. I didn't have time to try again.
I want to give it ago.
Should I post my results here or open a new issue? (Since this is closed).
Thanks!

EDIT: I saw the bot reopened, I'll add my findings soon. sry!

**Comment by RetroGameTalk** [comment]
Testing 4.2.2 vs 4.4.1 now.
I no longer get 400 Bad request, I get 200 OK.

But something is wrong.
1) i get errors on the new log: (this is **after delete everything and full resync**)

```
**Error**	
field [post_date_gmt] is of type [indexed,tokenized], but only numeric types are supported.
Recommended Solution	
It seems you saved a post without doing a full sync first because post_date_gmt is missing the correct mapping type. Delete all data and sync to fix the issue.
```


```
**Error**	
Fielddata is disabled on text fields by default. Set fielddata=true on [post_date] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory. Alternatively use a keyword field instead.
Recommended Solution	
It seems you saved a post without doing a full sync first because post_date is missing the correct mapping type. Delete all data and sync to fix the issue.
```

2) Facets widgets doesn't render any more. This is how i do it, let me know if it's wrong:
```
$args['before_widget'] = '<div class="facet-wrap">';
$args['after_widget'] = '</div>';
$args['before_title'] = '<span class="filter-title">';
$args['after_title'] = '</span>';
echo '<aside class="widget-area">';
echo '<section id="ep-facet-3" class="widget widget_ep-facet">';
$taxonomies = array( 'genre', 'language', 'region');
	
	foreach ($taxonomies as $tax) {
		$instance['facet'] = $tax;
		$instance['title'] = $tax.'s';
  the_widget( 'ElasticPress\Feature\Facets\Widget', $instance, $args );
  
	}
  echo '</section>
  </aside>';
    
    ?>
```
Please if you can ELI5 how to fix these cos I don't want to be stuck in an old version.
Thanks!

**Comment by felipeelia** [comment]
@MediaMaquina it seems you have a problem while you are sending the mapping. Do you have access to use WP-CLI? If so, do you mind running `wp elasticpress put-mapping` and sharing the output with us? If you are in a multisite, you will want to run `wp elasticpress put-mapping --network-wide`.

Also sharing the Status Report page contents will help us to identify potential problems. Thanks.

ps.: As you saw the issue will be automatically closed after a while but feel free to leave a comment. That will reopen it :)
